images/ubuntu-jammy/github_agent.ubuntu.pkr.hcl

```diff
@@ -168,6 +168,12 @@ build {
       "echo 'alias python=python3' >> /home/ubuntu/.bashrc",
       "echo 'alias pip=pip3' >> /home/ubuntu/.bashrc",
       "python3 -m pip install --upgrade pip",
+      "curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.5/install.sh | bash" # Node Version Manager
+      "nvm install 20" # Node 20
+      "npm install -g pnpm" # Package manager
+      "pnpm install -g wrangler" # Cloudflare cli
+      "pnpm install -g @sentry/cli@2.17.4" # Sentry cli
+      "npx playwright install-deps", # System dependencies for playwright e2e testing
```

Comment:
Added the system level dependencies we need for all frontend related checks. Having these built-in to the actions runner can help us save time setting all of these on every run.